byron/ledger/formal-spec/utxo.tex

```diff
@@ -114,24 +114,56 @@ \section{UTxO}
 \end{figure}
 
 \begin{figure}
+  \emph{UTxO environments}
+  \begin{equation*}
+    \UTxOEnv =
+    \left(
+      \begin{array}{r@{~\in~}lr}
+        \var{utxo_0} & \UTxO & \text{genesis UTxO}\\
+        \var{pps} & \PPMMap & \text{protocol parameters map}\\
+      \end{array}
+    \right)
+  \end{equation*}
+
   \emph{UTxO transitions}
   \begin{equation*}
     \_ \vdash
     \var{\_} \trans{utxo}{\_} \var{\_}
-    \subseteq \powerset (\PPMMap \times \UTxO \times \Tx \times \UTxO)
+    \subseteq \powerset (\UTxOEnv \times \UTxO \times \Tx \times \UTxO)
   \end{equation*}
   \caption{UTxO transition-system types}
   \label{fig:ts-types:utxo}
 \end{figure}
 
 \begin{figure}
+  \begin{equation}\label{eq:utxo-bootstrap}
```

Comment:
Thanks for updating the formal specs as well! Don't we need to update the executable specs also?